### ISSUE TITLE
flac.js fails for me silently

### ISSUE THREAD

**Issue by ddanila** [audiocogs/flac.js#18]
I can't play flac files with quite simple code:
var player = AV.Player.fromURL('http://example.com/file.flac');
player.play();

The very same code, but for mp3 works:
var player = AV.Player.fromURL('http://example.com/file.mp3');
player.play();

aurora.js 0.4.2 from https://github.com/audiocogs/aurora.js/releases
flac.js 0.2.1 from https://github.com/audiocogs/flac.js/releases

I see no errors in js console. Chrome Windows 8.1 Version 37.0.2062.120 m
the only suspicious things that I see are:
player.asset.active == false
player.playing == true
for flac

The flac that does not work is http://www.eclassical.com/custom/eclassical/files/BIS1536-001-flac_16.flac for example.

May be it's more relevant to report aurora.js issue instead, I don't know.

**Comment by master255** [comment]
@ddanila try to looking my work code http://habrahabr.ru/post/222729/

**Comment by ddanila** [comment]
@master255 
Your example works for me (I actually tried it before already and decided to try simplest solution to build my own UI instead of one you used).
I also tried flac file you used with my code -- does not work also.
I tried to use aurora.js and flac1.js from your example with my code -- does not work for me.
I assume that I am doing something wrong, but taking into the account that .mp3 works (and it uses external decoder -- mp3.js) with the same code, I can't imagine what can I do wrong with two lines of code

EDIT: Actually, your example does not work from my webserver with same symptoms, which probably means it's something with the webserver settings then. Will check.

EDIT2: AddType audio/flac .flac in .htaccess made the trick. Sorry, my fault, this issue can be closed then.
